Hindawi Publishers.js

```diff
@@ -80,7 +80,29 @@ function doWeb(doc,url)
 				// use article ID as a page (seems to be the last part of URL/DOI)
 				item.pages = 'e' + item.DOI.substr(item.DOI.lastIndexOf('/') + 1);
 			}
-			item.extra = "";
+			//remove duplicate metadata			
+			for (var i in item){
+				if (typeof item[i] == "string" && item[i].match(/^.+,/)){
```

Comment:
Was the `.match` part necessary? More curious than anything, since I can't think of a reason why it should be.